packages/lesswrong/server/vulcan-lib/apollo-ssr/renderPage.tsx

```diff
@@ -257,6 +258,15 @@ function maybeStartQueuedRequests() {
     let requestToStartRendering = requestPriorityQueue.dequeue();
     if (requestToStartRendering.request) {
       const { preOpPriority, request } = requestToStartRendering;
+      const { startTime, res } = request.renderRequestParams;
+      
+      const queuedRequestTimeoutSeconds = queuedRequestTimeoutSecondsSetting.get();
+      const maxRequestAge = moment().subtract(queuedRequestTimeoutSeconds, 'seconds').toDate();
```

Comment:
Nit (feel free to ignore): there's no reason to use `moment` here; this could just be
```
const requestAgeMs = new Date().getTime() - startTime.getTime();
if (requestAgeMs > queuedRequestTimeoutSeconds * 1000)
```